SkietbaanBE/SkietbaanBE/Controllers/ScoresController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using SkietbaanBE.Models;
+using SkietbaanBE.RequestModel;
+
+namespace SkietbaanBE.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/Scores")]
+    public class ScoresController : Controller
+    {
+        private ModelsContext _context;
+        public ScoresController(ModelsContext context)
+        {
+            _context = context;
+        }
+        [HttpGet]
+        [HttpPost]
+        public IActionResult ScoreCapture([FromBody]ScoreCapture scoreCapture)
+        {
+            if (ModelState.IsValid)
+            {
+                Competition competition = _context.Competitions.Where(x => x.Name == scoreCapture.CompetitionName).FirstOrDefault<Competition>();
```

Comment:
You don't check competition or user for null, might be useful to return some 404's if they are null

Reply:
Also make competition and user a `var`, it makes code updates easier later on

Reply:
ok changed